ci-operator/config/cri-o/cri-o/cri-o-cri-o-release-1.13.yaml

```diff
@@ -0,0 +1,64 @@
+base_images:
+  base:
+    name: "4.1"
```

Comment:
4.2?

Reply:
This is for cri-o 1.13, so that should be for 4.1. cri-o 1.14 will be for 4.2.